docs/gitbook/architecture/permission_model.adoc

```diff
@@ -0,0 +1,186 @@
+blockchains (which is a key part of how <<chain_joins,chain joins>>
+work).
+
+
+[[permissions]]
+=== Permissions
+
+Permissions are the specific actions that your application supports
+that can be assigned to roles.
+
+For example, in a voting application, the `add_vote` action which
+records a single vote on a blockchain could be assigned to the
+`can_vote` role, permitting any user with the `can_vote` role to
+dispatch the `add_vote` action.
+
+Each Interbit application includes a special permission called `*` (the
+_star_ permission). The `*` means _any action_, and is typically
+assigned only to the `root` role. In addition, the `*` can be used as a
+wildcard for defining permissions. For example, a `USER_*` permission
+would match all actions that start with `USER_`.
+
+Also, there are typically two permissions for managing the ACL itself:
+`addToAcl` and `replaceAcl`. These are included in addition to any
+actions that your Interbit application implements.
+
+[WARNING]
+=========
+You must be careful when adjusting permissions. Removing your own
+permissions prevents you from performing actions. Removing the
+permission to adjust the ACL means that nobody can change the
+blockchain's permission in the future. Removing the `*` permission could
+render your blockchain **permanently unusable**.
+=========
+
+[NOTE]
+======
+At this time, all users with access to a blockchain can read from the
+blockchain. Future versions of the Interbit framework will provide the
+ability to control read access.
+======
+
+
+[[action_dispatch]]
+== Action Dispatch
+
+When an action is dispatched to a blockchain, the following operations
+are performed:
+
+- The application constructs a _payload_ that represents all of the
+  information required to successfully complete the action.
+
+- The payload is then combined with its cryptographic hash and the
+  result is signed with the user's private key. This makes the action
+  _verifiable_.
+
+- The blockchain's smart contract then checks the signature to verify
+  that the user has access.
+
+- The blockchain's smart contract then compares the payload to the hash
+  to verify that the payload has not been modified. This makes actions
+  _tamper resistant_.
+
+- The blockchain's smart contract then evaluates the ACL to determine
+  whether the user has the permission to perform the action.
+
+If all of these steps complete successfully, the smart contract performs
+the action and updates the blockchain accordingly, and propagates the
+changes to all participating nodes in the blockchain's network.
+
+
+[[chain_joins]]
+== Chain Joins
+
+Chain joins are an Interbit-specific feature. They permit the
+information in one blockchain to be connected with information in
+another blockchain. There are two kinds of chain joins: read joins and
+write joins.
+
+A read join means that one blockchain, the _provider_, has access to a
+copy of the state -- the collection of data stored on the blockchain --
```

Comment:
We should be clear that this need not be _all_ of state. 

This has been a common misunderstanding.